src/classifier/Parameters.h

```diff
@@ -24,12 +24,88 @@
 #include "SparseLabeledParameterVector.h"
 #include "Utils.h"
 
+
+
+
+
+
+//struct  FeatureLabelPair : Structure to define a a feature-label pair
```

Comment:
Replace this comment by:

// Structure to define a feature-label pair.

i.e.:
- 1 space after //
- no need to repeat "struct FeatureLabelPair"
- end with punctuation.